pf/tfbridge/main.go

```diff
@@ -54,6 +55,22 @@ func Main(ctx context.Context, pkg string, prov tfbridge.ProviderInfo, meta Prov
 	}
 }
 
+func handleGetSchemaFlag(prov tfbridge.ProviderInfo) {
+	flags := flag.NewFlagSet("get-schema-flags", flag.ContinueOnError)
+
+	dumpSchema := flags.Bool("get-schema", false, "dump provider schema as JSON to stdout")
+
+	flags.SetOutput(io.Discard)
+
+	err := flags.Parse(os.Args[1:])
+	contract.IgnoreError(err)
+
```

Comment:
Why does PF need this ceremony when pkg can just call `flags.Bool`?
```go
	dumpSchema := flags.Bool("get-schema", false, "dump provider schema as JSON to stdout")

```

Reply:
handleFlags takes a MarshallableProviderInfo, which does not have the required fields for DetailedSchemaDump